app/utils/helpers.ts

```diff
@@ -95,6 +95,19 @@ export function formatDate(dateString: string | Date, locale = 'de-DE') {
   const month = date.toLocaleString(locale, { month: 'long' });
   const year = date.toLocaleString(locale, { year: 'numeric' });
 
+  return `${day}. ${month} ${year}`;
+}
+
+export function formatDateWithTimestamp(dateString: string | Date, locale = 'de-DE') {
+  if (!dateString || isNaN(new Date(dateString).getTime())) {
+    return null;
+  }
+
+  const date = new Date(dateString);
+  const day = date.toLocaleString(locale, { day: 'numeric' });
+  const month = date.toLocaleString(locale, { month: 'long' });
+  const year = date.toLocaleString(locale, { year: 'numeric' });
+
   const hour = date.getHours().toString().padStart(2, '0');
   const minute = date.getMinutes().toString().padStart(2, '0');
```

Comment:
is the .padStart(2,'0') needed?

Reply:
In my opinion yes, without padStart the time says 9:27 instead of 09:27 and 13:1 instead of 13:01. I think it looks better with the "0" in front of both hours and minutes